### PR TITLE
YAML syntax for ansible tasks

### PR DESCRIPTION
**Pull request by arbabnazar**
@edx/devops kindly review. Thanks

### CODE REVIEW

File: playbooks/roles/ecommerce/tasks/main.yml [hunk 1]
```diff
@@ -20,48 +20,52 @@
...
-    . {{ ecommerce_nodeenv_bin }}/activate && {{ ecommerce_node_bin }}/bower install --production --config.interactive=false
+- name: Install bower dependencies
+  shell: ". {{ ecommerce_nodeenv_bin }}/activate && {{ ecommerce_node_bin }}/bower install --production --config.interactive=false"
+  args:
+    chdir: "{{ ecommerce_code_dir }}"
   become_user: "{{ ecommerce_user }}"
 
-- name: migrate
+- name: Migrate
   shell: >
```
Comment (jibsheet) [inline playbooks/roles/ecommerce/tasks/main.yml:58]: Missed > - oh - I see, whole thing is still a block now

**Comment by jibsheet** [comment]
One missed block - can you rebase this and build an AMI, Michael also landed some ecommerce play changes.

File: playbooks/roles/ecommerce/tasks/main.yml (hunk 1 above)
Comment (arbabnazar) [inline playbooks/roles/ecommerce/tasks/main.yml:58]: @jibsheet sorry I didn't get it.

**Comment by arbabnazar** [comment]
@jibsheet Built the AMI successfully with this branch using alton. Kindly review. Thanks

Job reference: 4940

CC: @edx/devops

**Comment by jibsheet** [comment]
👍 
@MichaelRoytman I believe you'll need to rebase your PR when this merges.

**Comment by arbabnazar** [comment]
Thanks @jibsheet 

@MichaelRoytman please let me know if you need any help

**Pull request merged**